spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -35,8 +35,10 @@ In particular:
 
 - The `jakarta.nosql.Entity` annotation from the Jakarta NoSQL specification is an entity-defining annotation for Jakarta Data providers backed by NoSQL databases. Other Jakarta Data providers must not support the use of `jakarta.nosql.Entity` as an entity-defining annotation.
 
+
 A Jakarta Data provider might define a custom entity-defining annotation.
-The name of such a custom entity-defining annotation type must end with `Entity`. This convention allows any Jakarta Data provider to identify when a repository is associated with entity classes declared using an unrecognized entity-defining annotation.
+
+To ensure interoperability within the Jakarta Data ecosystem, custom entity-defining annotations must be marked with `EntityQualifier`. This approach ensures that Jakarta Data providers can recognize and process entities declared with custom annotations, thereby maintaining consistency and flexibility in data management.
```

Comment:
@otaviojava I know I'm being a bit annoying here, but t's enough to just state the requirements. It's a spec, not a whitepaper.

```suggestion
Custom entity-defining annotations must be marked with `@EntityQualifier`. 
This allows other Jakarta Data providers and tools to recognize and process entities declared with custom entity-defining annotations.
```

Reply:
Nope, you are not annoying.
I appreciate your patience.